block/manager.go

```diff
@@ -657,18 +682,69 @@ func (m *Manager) publishBlockInternal(ctx context.Context) error {
 	}
 
 	m.recordMetrics(data)
-
 	g, ctx := errgroup.WithContext(ctx)
 	g.Go(func() error { return m.headerBroadcaster.WriteToStoreAndBroadcast(ctx, header) })
 	g.Go(func() error { return m.dataBroadcaster.WriteToStoreAndBroadcast(ctx, data) })
 	if err := g.Wait(); err != nil {
 		return err
 	}
 
-	m.logger.Debug("successfully proposed header", "proposer", hex.EncodeToString(header.ProposerAddress), "height", headerHeight)
 	return nil
 }
 
+// previousBlockData holds information about the previous block,
+// necessary for creating the next block.
+type previousBlockData struct {
```

Comment:
I understand that this represents historic block data but the term "previous" feels a bit too technical to the process. I know naming is hard but maybe you can find a term that is more focused on what this data represents or how it is used?